lib/checkleakautovar.h

```diff
@@ -93,8 +93,6 @@ class CPPCHECKLIB VarInfo {
 
     /** set possible usage for all variables */
     void possibleUsageAll(const std::pair<const Token*, Usage>& functionUsage);
-
-    void print();
```

Comment:
It looks like this was never used since its introduction in bd8fb0a6b5b85b1efe5f83cae3d490bd3c402c0d.